source/features/clean-dashboard.css

```diff
@@ -12,21 +12,30 @@
 	display: none;
 }
 
-/* Add background and border Whole list */
-.rgh-clean-dashboard .news > .f4 + div {
-	background-color: var(--rgh-background);
-	border: 1px solid var(--rgh-border-color);
-	border-radius: 6px;
-	margin-top: 8px !important;
-}
-
-/* Drop "box" style for all events */
-.rgh-clean-dashboard .news > .f4 + div .Box {
```

Comment:
It feels like this specific sequence was made to handle event groups. Do you see any issues with my hotfix on your dashboard?

Reply:
Groups of multiple releases and multiple commits are looking good for me :+1:

Reply:
![image](https://user-images.githubusercontent.com/46634000/159501103-e541dcfe-0205-4f55-8e55-47c8339bafbe.png)

This could be compressed a bit more by dropping some of the `<hr>` and reducing the vertical spacing before & after the reactions buttons (or dropping them altogether)

Reply:
Releases are normally hidden by refined GitHub, so can’t do that 😃